UPGRADING-HADOOP.md

```diff
@@ -0,0 +1,181 @@
+# Hadoop Upgrade
+## Actions and Instructions
+
+The hadoop-base layer contains an action called hadoop-upgrade which allows the
+administrator to upgrade, downgrade and finalize a hadoop deployment. Currently
+rollback is not supported, and HDFS filesystem version upgrade is not supported (but
+will be implemented when a new filesystem version is released greater than the version
+which comes with 2.7.2)
+
+The actions and the commands they run are in line with the instructions and
+recommendations in this document:
+
+http://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-HDFS/HDFSRollingUpgrade.html
```

Comment:
This URL was giving me a 404. This must be what you were refering to:
http://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-hdfs/HdfsRollingUpgrade.html